applications/FluidDynamicsApplication/custom_elements/fluid_adjoint_element.cpp

```diff
@@ -0,0 +1,800 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/checks.h"
+#include "includes/element.h"
+#include "includes/properties.h"
+#include "utilities/adjoint_extensions.h"
+#include "utilities/geometrical_sensitivity_utility.h"
+#include "utilities/indirect_scalar.h"
+
+// Application includes
+#include "custom_elements/data_containers/fluid_adjoint_derivatives.h"
+#include "fluid_dynamics_application_variables.h"
+
+#include "custom_elements/data_containers/qs_vms/qs_vms_adjoint_element_data.h"
+
+// Include base h
+#include "fluid_adjoint_element.h"
+
+namespace Kratos
+{
+namespace FluidAdjointElementHelperUtilities
```

Comment:
I'd add this as a static method of an auxiliary class to avoid the creation of a new namespace.

Reply:
These are helper functions in a cpp which cannot be used anywhere else. Would there be a special gain or loss by converting them to static methods?

Reply:
If only used here, then I'd make an anonymous namespace.